mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -255,22 +255,30 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
 	METHOD method_30120 (Ljava/util/List;Lnet/minecraft/class_1304;Lnet/minecraft/class_1799;)V
 		ARG 2 slot
 		ARG 3 stack
-	METHOD method_30121 swapHandStacks (Ljava/util/Map;)V
-		ARG 1 equipment
-	METHOD method_30122 setArmorInSlot (Lnet/minecraft/class_1304;Lnet/minecraft/class_1799;)V
+	METHOD method_30121 sendHandStackSwap (Ljava/util/Map;)V
```

Comment:
I prefer
```suggestion
	METHOD method_30121 checkHandStackSwap (Ljava/util/Map;)V
```
as this first checks and sends the swap only if the hands have updated